src/ubuntu/build-scripts/build-one-rootfs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Stop script on NZEC
+set -e
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
+
+os=$1
+crossToolset=$2
+arch=$3
+lldb=$4
+
+scriptsVolume="scripts$(date +%s)"
+dockerCrossDepsTag="microsoft/dotnet-buildtools-prereqs:${os}-crossdeps"
+
+echo "Using $dockerCrossDepsTag to clone core-setup to fetch scripts used to build cross-toolset"
+docker run --rm -v $scriptsVolume:/scripts $dockerCrossDepsTag git clone https://github.com/dotnet/core-setup /scripts
+
+rm -rf $PWD/rootfs
+mkdir rootfs
+
+echo "Using $dockerCrossDepsTag to set up cross-toolset for $arch for $crossToolset"
+buildRootFSContainer="rootfs-$arch-$crossToolset"
+docker run --privileged --rm --name $buildRootFSContainer -e ROOTFS_DIR=/rootfs/$arch \
```

Comment:
While it is not a lot of logic, I would prefer to not duplicate this.  What do you think about adding optional parameters to build-rootfs so that a single script can serve both scenarios?  If the $arch isn't passed in, it will default to the set in build-rootfs today?

Reply:
My idea was to duplicate it until the original file was deleted because it was no longer needed.

I'll revise to share the file

Reply:
That's a fair point - my primary concern is the #16 actually gets implemented in a timely fashion.  I would be fine with the duplication in the interim.